scripts/dream.py

```diff
@@ -167,6 +177,23 @@ def main_loop(t2i, outdir, parser, infile):
             if not os.path.exists(opt.outdir):
                 os.makedirs(opt.outdir)
             current_outdir = opt.outdir
+        elif prompt_as_dir:
+            if prompt_as_dir == 'simple':
+                # this line will sanitize the entire prompt as one folder
+                subdir = path_filter.sub('_', opt.prompt)[:name_max].rstrip(' .')
+            else:
+                # this line will allow the prompt to create subdirectories
+                subdir = os.path.join(*[path_filter.sub('_', x)[:name_max].rstrip(' .') for x in opt.prompt.replace('\\', '/').split('/') if x.rstrip(' .') != ''])
+
+            # truncate path to maximum allowed length
+            subdir = subdir[:(path_max - 27 - len(os.path.abspath(outdir)))]
```

Comment:
Where is `27` coming from?

Reply:
Addressed in a previous pr comment, it is the length of '######.##########.##.png', plus two directory separators and a nul terminator.